ci/scripts/stage.sh

```diff
@@ -37,7 +37,7 @@ git commit -m "Release v$stageVersion"
 git tag -a "v$stageVersion" -m "Release v$stageVersion"
 echo
 
-./gradlew --no-daemon clean install -Dmaven.repo.local="${repository}"
+./gradlew clean publish -PpublicationRepository="${repository}"
```

Comment:
Gradle's recommendation since 3.0 is to [use the daemon in CI](https://docs.gradle.org/current/userguide/gradle_daemon.html#sec:disabling_the_daemon) unless for some reason it's causing instability.

Reply:
thanks for the explanation!